regimes/pt/README.md

```diff
@@ -15,7 +15,7 @@ Find example PT GOBL files in the [`examples`](../../examples/pt) (uncalculated
 
 ### `InvoiceType` (Tipo de documento)
 
-SAF-T's `InvoiceType` (Tipo de documento) specifies the type of a Portuguese tax document. The following table lists all the supported invoice types and how GOBL will map them with a combination of invoice type and tax tags:
+SAF-T's `InvoiceType` (Tipo de documento) specifies the type of a sales invoice. In GOBL, this type can be set using the `pt-saft-invoice-type` extension in the tax section. GOBL will set the extension for you based on the type and the tax tags you set in your invoice. The table below shows how this mapping is done:
```

Comment:
Could all of this README stuff be added to the Addon description and extensions like for GR now? This way it'll get published automatically to the documentation site. I understand there are also specific for the addon, and probably shouldn't be in the Regime.

Reply:
Cool, I'll move PT taking GR as a reference

Reply:
Addressed in [43c18e3](https://github.com/invopop/gobl/pull/457/commits/43c18e37a7df7c17d49d32e07bf21e9d05f7d395)